src/main/java/cceclipseplugin/core/EclipseRequestManager.java

```diff
@@ -124,36 +132,139 @@ public void pullFileAndCreate(IProject p, Project ccp, File file, IProgressMonit
 						}
 						fileContents = PluginManager.getInstance().getDataManager().getPatchManager().applyPatch(fileContents, patches);
 						if (newFile.exists()) {
-							newFile.setContents(new ByteArrayInputStream(fileContents.getBytes()), true, false, progressMonitor);
+							ByteArrayInputStream in = new ByteArrayInputStream(fileBytes);
+							newFile.setContents(in, false, false, progressMonitor);
+							
+							in.close();
```

Comment:
I think some amount of these would be better as a try-with-resources statement, so that it would handle the closing for us.